Formula/chatblade.rb

```diff
@@ -0,0 +1,130 @@
+class Chatblade < Formula
+  include Language::Python::Virtualenv
+
+  desc "CLI Swiss Army Knife for ChatGPT"
+  homepage "https://github.com/npiv/chatblade"
+  url "https://files.pythonhosted.org/packages/23/b9/31ee0c050069c212d09027129e37fb0e5392c1827770bfe046977cac9340/chatblade-0.2.3.tar.gz"
+  sha256 "61765ea885fc731343bcce0f9f4a2be8baa3d9f7d7f5a41bddbff13087811d19"
+  license "GPL-3.0-only"
+
+  depends_on "rust" => :build
```

Comment:
Why do you need Rust?

Reply:
This dependency: https://github.com/openai/tiktoken